app/services/slack_service.rb

```diff
@@ -0,0 +1,29 @@
+module SlackService
+  extend self
+
+  OPEN_SOURCE = '#open-source'.freeze
+  ENGINEERING_METRICS_URL = ENV['ENGINEERING_METRICS_URL']
+  EMOJI = ':wave:'.freeze
+
+  def open_source_reminder
+    message = I18n.t('services.slack.open_source_reminder.message',
+                     url: "#{ENGINEERING_METRICS_URL}/open_source")
+    ping(OPEN_SOURCE, message)
+  rescue Slack::Notifier::APIError => exception
+    ExceptionHunter.track(exception)
```

Comment:
doesn't ExceptionHunter track things automatically? cc @brunvez

Reply:
yess, it should track them on jobs automatically, giving more context than doing it manually